src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -107,6 +175,20 @@ class PackageDetailTab extends mixin(StoreMixin) {
     this.setState({ openInstallModal: true, advancedConfig: true });
   }
 
+  handlePackageVersionChange(selection) {
```

Comment:
This might be a stupid questions, but why do we need to "manually" redirect users? Can't we simply provide a list of links which will in turn trigger a rout update?

Reply:
We tried that but since we have no route context access the link did not work.

Reply:
I'm not sure I understand the issue. Could you please elaborate?

Reply:
@orlandohohmeier our dropdown's render outside of the React context using https://github.com/mesosphere/reactjs-components/blob/master/src/Portal/Portal.js which means a `<Link />` does not work.